src/price_data.py

```diff
@@ -468,7 +468,9 @@ def _get_price_kraken(
 
                 if not data["error"]:
                     break
-                elif data["error"] == ["EGeneral:Invalid arguments"]:
+                elif data["error"] == ["EGeneral:Invalid arguments"] or data[
+                    "error"
+                ] == ["EQuery:Unknown asset pair"]:
```

Comment:
Ugly formatting, but this is how `black` autoformats

Reply:
What happens, if you put brackets around like this and use black to autoformat `((a) or (b))`?

Reply:
thanks, looks better now